editors/code/src/client.ts

```diff
@@ -7,14 +7,28 @@ import { WorkspaceEdit } from "vscode";
 import { Workspace } from "./ctx";
 import { updateConfig } from "./config";
 import { substituteVariablesInEnv } from "./config";
+import { randomUUID } from "crypto";
 
 export interface Env {
     [name: string]: string;
 }
 
-function renderCommand(cmd: ra.CommandLink) {
-    return `[${cmd.title}](command:${cmd.command}?${encodeURIComponent(
-        JSON.stringify(cmd.arguments)
+export const LINKED_COMMANDS = new Map<string, ra.CommandLink>();
```

Comment:
I think it would be good to document this workaround in-tree, so could you copy part of the PR description here as an inline comment?